python/perspective/perspective/tests/table/test_table_arrow.py

```diff
@@ -22,14 +23,22 @@
 
 
 class TestTableArrow(object):
-
     def test_table_arrow_loads(self):
-        with open(SUPERSTORE_ARROW, mode='rb') as file:  # b is important -> binary
+        if not os.path.exists(SUPERSTORE_ARROW):
+            file_path = SUPERSTORE_ARROW_MINI
+            size = 10
+            width = 19
+        else:
+            file_path = SUPERSTORE_ARROW
+            size = 9994
+            width = 21
+
+        with open(file_path, mode='rb') as file:  # b is important -> binary
```

Comment:
This is because `yarn install` does not run on python builds?  It looks like in the `build.yml` it does though.

I think there are better ways to avoid this - let's either find a way to get `superstore.arrow` in the python build (as we may already and this is debug code again), or just use a small superstore-mini ... though I really would prefer we re-use the asset we published to NPM explicitly for this purpose.

Reply:
I don't think python tests should require a developer installation (e.g. `../../../../node_modules` isn't going to work on somebody who just downloads perspective-python) so i've eliminated the node_modules superstore in favor of the local one

Reply:
Running tests from a e.g. pypi install of `perspective-python` is not a design goal.

Reply:
It should be, this is how e.g. conda would test that it was built properly, how how one might test the compatibility of their perspective installation with the rest of their install